ui/src/views/Tasks/InteractiveConnect/index.jsx

```diff
@@ -300,6 +334,18 @@ export default class InteractiveConnect extends Component {
               secondary={<StatusLabel state={interactiveStatus} />}
             />
           </ListItem>
+          <ListItem>
+            <FormControlLabel
+              control={
+                <Switch
+                  checked={notificationToggle}
+                  onChange={this.handleNotificationChange}
+                  color="secondary"
+                />
+              }
+              label="Notify Me on Ready"
+            />
+          </ListItem>
           <ListItem
             button
             className={classNames(
```

Comment:
I made some changes. You can look at the diff in https://github.com/helfi92/taskcluster/commit/262fc0a51bba4d29b54880dc6182c6497077f64d or simply copy paste [this file](https://raw.githubusercontent.com/helfi92/taskcluster/262fc0a51bba4d29b54880dc6182c6497077f64d/ui/src/views/Tasks/InteractiveConnect/index.jsx). Let me know if there's anything unclear.